docs/en/sql-reference/statements/select/except.md

```diff
@@ -146,8 +146,73 @@ Result:
 └─────────────┘
 ```
 
+# EXCEPT Column(s)
 
-**See Also**
+`EXCEPT` can also be used to remove unwanted columns, especially used with a sub query (e.g. `row_number()`).
+
+## Example
+
+The example starts with a table with 2 columns:
+
+```sql
+CREATE TABLE test1  (
+    name String,
+    created_at Datetime
+) ENGINE=MergeTree ORDER BY (name);
+```
+
+Prepare some data:
+
+```sql
+INSERT INTO test1 VALUES ('Felix', NOW()), ('Bob', NOW());
+```
+
+To show the basic usage, you can select all fields except `created_at`:
+
+```sql
+SELECT * EXCEPT(created_at) FROM test1
+```
```

Comment:
Please include the result of the SELECT in l. 173 here.

Reply:
Done. PTAL.